README.md

```diff
@@ -1,95 +1,120 @@
+
+## Prerequisites
+
+* [Carthage](https://github.com/Carthage/Carthage): Can be installed with [Homebrew](http://brew.sh/).
 
-1. User opens up the app in iOS based mobile phone and chooses the different ways (faces, explicit, food etc.) they can classify images.
-2. Visual Recognition service in IBM cloud classifies and provides with the classification results.
+  ```bash
+  brew install carthage
+  ```
+
+* [Xcode](https://developer.apple.com/xcode/): Required to develop for iOS, can be found on the [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835?mt=12).
+
+* [IBM Cloud](https://cloud.ibm.com) account.
+
+> **Note**: :bulb: You can use the free/lite tier of IBM Cloud to use this code!
 
 ## Steps
 
 1. [Clone the repo](#1-clone-the-repo)
-2. [Carthage dependency installation](#2-carthage-dependency-installation)
+2. [Install dependencies with Carthage](#2-install-dependencies-with-carthage)
 3. [Setup Visual Recognition credentials](#3-setup-visual-recognition-credentials)
-4. [Install Xcode and run the App](#4-install-xcode-and-run-the-app)
+4. [Run the app with Xcode](#4-run-the-app-with-xcode)
 
 ### 1. Clone the repo
+
 `git clone` the repo and `cd` into it by running the following command:
 
 ```bash
 git clone github.com/IBM/watson-visual-recognition-ios.git &&
 cd watson-visual-recognition-ios
 ```
 
-### 2. Carthage dependency installation
-You can install Carthage with [Homebrew](http://brew.sh/):
-
-```bash
-brew install carthage
-```
-
-If your project does not have a Cartfile yet, generate a Cartfile with the Watson Developer Cloud Swift SDK: 
-
-```
-echo 'github "watson-developer-cloud/swift-sdk" ~> 1.4.0' > Cartfile
-```
+### 2. Install dependencies with Carthage
 
 Then run the following command to build the dependencies and frameworks:
 
 ```bash
 carthage update --platform iOS
 ```
 
-### 3. Setup Visual Recognition credentials
+> **Tip**: :bulb: This step could take some time, please be patient.
+
+### 3. Create IBM Cloud services
+
+Create the following services:
 
-Go to [IBM Cloud](https://cloud.ibm.com) and sign up if you haven't done so. Search for `Visual Recognition` in catalog search and create one using the lite plan. Copy the `apikey` from the credentials to use it for later.
+  * [**Watson Visual Recognition**](https://cloud.ibm.com/catalog/services/visual-recognition)
+
+Copy the *API Key* from the credentials and add it to [`Credentials.plist`](https://github.com/IBM/watson-visual-recognition-ios/blob/master/Core%20ML%20Vision/Credentials.plist)
 
 ![Credentials](docs/images/vizrec-creds.gif)
 
-Add your Visual Recognition API key to the [`Credentials.plist`](https://github.com/IBM/watson-visual-recognition-ios/blob/master/Core%20ML%20Vision/Credentials.plist):
+#### [`Credentials.plist`](https://github.com/IBM/watson-visual-recognition-ios/blob/master/Core%20ML%20Vision/Credentials.plist)
+
 ```plist
 <key>apiKey</key>
-<string>YOUR_API_KEY</string>
+<string>Rnqw721s...0m8aPu</string>
```

Comment:
should this be API_KEY?

Reply:
🤷‍♂ i like giving the user an idea of what to expect for a key, i don't think it matters much. If you want it back the other way, I can revert.

Reply:
I was going to ask the same

Reply:
I am fine with both.

Reply:
I've had a lot of people blindly copy exactly what I put in the examples, so I like to use as verbose of a "REPLACE_ME" as possible